src/extension.ts

```diff
@@ -63,3 +86,76 @@ const getComponentName = (): Thenable<string | undefined> =>
         : "Please enter a valid component name";
     },
   });
+
+/**
+ * Check availability of Angular CLI on the command line
+ * @param directory Directory which will use the Angular CLI
+ */
+const checkAngularCli = (directory: string) => {
```

Comment:
(Optional) same as with `getDirectoryName()`. You should return `false` if its not available and log the error instead. This simplifies the caller code to:

```ts
const useNpx = isAngularCliAvailable(componentDirectory);
```

Reply:
Feels better like this :)